src/rdb/rdb_util.c

```diff
@@ -259,7 +259,7 @@ rdb_vos_tx_end(struct rdb *db, rdb_vos_tx_t vtx, int err)
 	int                rc;
 
 	rc = dtx_end(dth, NULL /* cont */, err);
-	if (rc != 0)
+	if (rc != err)
```

Comment:
[Note] This is to avoid logging an `ERR` when we pass `-DER_AGAIN` to abort and later retry a vtx.